ecosystem/python/sdk/aptos_sdk/transactions.py

```diff
@@ -128,30 +175,35 @@ def __init__(
         self.raw_transaction = raw_transaction
         self.secondary_signers = secondary_signers
 
-    def inner(self) -> RawTransaction:
-        return self.raw_transaction
-
-    def prehash(self) -> bytes:
-        hasher = hashlib.sha3_256()
-        hasher.update(b"APTOS::RawTransactionWithData")
-        return hasher.digest()
-
-    def keyed(self) -> bytes:
-        serializer = Serializer()
+    def serialize(self, serializer: Serializer):
         # This is a type indicator for an enum
         serializer.u8(0)
         serializer.struct(self.raw_transaction)
         serializer.sequence(self.secondary_signers, Serializer.struct)
 
-        prehash = bytearray(self.prehash())
-        prehash.extend(serializer.output())
-        return bytes(prehash)
 
-    def sign(self, key: ed25519.PrivateKey) -> ed25519.Signature:
-        return key.sign(self.keyed())
+class FeePayerRawTransaction(RawTransactionWithData):
+    secondary_signers: List[AccountAddress]
+    fee_payer: Optional[AccountAddress]
+
+    def __init__(
+        self,
+        raw_transaction: RawTransaction,
+        secondary_signers: List[AccountAddress],
+        fee_payer: Optional[AccountAddress],
+    ):
+        self.raw_transaction = raw_transaction
+        self.secondary_signers = secondary_signers
+        self.fee_payer = fee_payer
 
-    def verify(self, key: ed25519.PublicKey, signature: ed25519.Signature) -> bool:
-        return key.verify(self.keyed(), signature)
+    def serialize(self, serializer: Serializer):
+        serializer.u8(1)
+        serializer.struct(self.raw_transaction)
+        serializer.sequence(self.secondary_signers, Serializer.struct)
+        fee_payer = (
+            AccountAddress.from_str("0x0") if self.fee_payer is None else self.fee_payer
```

Comment:
May be nice to provide these kinds of addresses as constants.

Reply:
hmm... zero :)